decoder/pokemon.go

```diff
@@ -449,7 +445,7 @@ func (pokemon *Pokemon) updateFromWild(ctx context.Context, db db.DbDetails, wil
 	if pokemon.setPokemonDisplay(int16(wildPokemon.Pokemon.PokemonId), wildPokemon.Pokemon.PokemonDisplay) {
 		updateStats(ctx, db, pokemon.Id, stats_statsReset)
 	}
-	pokemon.addWildPokemon(ctx, db, wildPokemon, timestampMs, true)
+	pokemon.addWildPokemon(ctx, db, wildPokemon, timestampMs)
```

Comment:
This will still call `spawnpointUpdate`. Maybe we can move that part of the functionality into `updatePokemonFromEncounterProto`?